.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,250 @@
+          file: 'neofs-lens-amd64'
+          target: 'neofs-testcases/neofs-lens'
+
+      - name: Download latest stable neofs-node
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neofs-node'
+          version: 'tags/v0.39.2'
+          file: 'neofs-node-amd64'
+          target: 'neofs-testcases/neofs-node'
+
+      - name: Checkout neofs-s3-gw repository
+        uses: actions/checkout@v4
+        with:
+          repository: nspcc-dev/neofs-s3-gw
+          ref: '014a5bf493d00b8cce478e2c96c84ad9de8fe617'
+          path: neofs-s3-gw
+
+      - name: Download latest stable neofs-rest-gw
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neofs-rest-gw'
+          version: 'tags/v0.6.0'
+          file: 'neofs-rest-gw-linux-amd64'
+          target: 'neofs-testcases/neofs-rest-gw'
+
+      - name: Download latest stable neofs-http-gw
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neofs-http-gw'
+          version: 'tags/v0.28.0'
+          file: 'neofs-http-gw-linux-amd64'
+          target: 'neofs-testcases/neofs-http-gw'
+
+      - name: Download latest stable neo-go
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neo-go'
+          version: 'tags/v0.104.0'
+          file: 'neo-go-linux-amd64'
+          target: 'neofs-testcases/neo-go'
+
+      - name: Chmod latest stable binaries
+        run: |
+          sudo chmod a+x neofs-cli
+          sudo chmod a+x neofs-adm
+          sudo chmod a+x neofs-ir
+          sudo chmod a+x neofs-lens
+          sudo chmod a+x neofs-node
+          sudo chmod a+x neofs-rest-gw
+          sudo chmod a+x neofs-http-gw
+          sudo chmod a+x neo-go
+        working-directory: neofs-testcases
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          cache: true
+          go-version: '1.20'
+      - run: go version
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10.11'
+      - run: python --version
+
+# Hashlib uses OpenSSL for ripemd160 and apparently OpenSSL disabled some older crypto algos around version 3.0
+# in November 2021. All the functions are still there but require manual enabling.
+# See https://github.com/openssl/openssl/issues/16994
+# But we use ripemd160 for tests.
+# For ripemd160 to be supported, we need the openssl configuration file to contain the following lines:
+# openssl_conf = openssl_init
+#
+# [openssl_init]
+# providers = provider_sect
+#
+# [provider_sect]
+# default = default_sect
+# legacy = legacy_sect
+#
+# [default_sect]
+# activate = 1
+#
+# [legacy_sect]
+# activate = 1
+      - name: Fix OpenSSL ripemd160
+        run: |
+          sudo python ./tools/src/openssl_config_fix.py
+        working-directory: neofs-testcases
+
+      - name: Build neofs-s3-gw
```

Comment:
Why not make a binary download here?

Reply:
I need a specific commit here, not a release version